sample/www/jxcore/thali/readme.md

```diff
@@ -0,0 +1,17 @@
+# Thali Cordova Plugin Replication Manager #
+
+### Contributing
+
+If you see a mistake, find a bug, or you think there is a better way to do something, feel free to contribute.
+Email [thali-talk@thaliproject.org](mailto:thali-talk@thaliproject.org) to connect with other contributors and
```

Comment:
Unfortunately the mailing list doesn't take emails from people who aren't registered. I would instead suggest telling them to file a Github issue.